netdata-installer.sh

```diff
@@ -1318,11 +1318,30 @@ should_install_ebpf() {
   return 0
 }
 
+remove_old_ebpf() {
+  if [ -f "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" ]; then
+    echo >&2 "Removing alpha eBPF collector."
+    rm -rf "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" 
```

Comment:
do we need `-r` here just to be on safe side because we know it is single file not directory?

Reply:
You are right, it was only a safe option.

Reply:
well removing `-r` would maybe help us if something funky is in ${NETDATA_PREFIX} ?

Reply:
You are right, I created a new commit that removes the `-r`.